resources/dicts/thoughts/ondeath/darkforest/leader_life.json

```diff
@@ -2,7 +2,10 @@
 	{
 		"id": "leader_life_dark_forest_thoughts",
 		"thoughts": [
-			"Was startled to find {PRONOUN/m_c/self} in the Dark Forest for a moment... did {PRONOUN/m_c/subject} just lose a life?"
+			"Was startled to find {PRONOUN/m_c/self} in the Dark Forest for a moment... did {PRONOUN/m_c/subject} just lose a life?",
+			"Has been somber ever since {PRONOUN/m_c/subject} lost a life...",
+			"Has been thinking of the cat who gave {PRONOUN/m_c/object} the life {PRONOUN/m_c/subject} just lost...",
+			"Is feeling a sense of dread. If {PRONOUN/m_c/subject} ended up in the Dark Forest... what kind of legacy are {PRONOUN/m_c/subject} creating?"
```

Comment:
verb tag 'are'
reads as both 'what kind of legacy are they creating?' and 'what kind of legacy are she creating?'

Reply:
I think I got it but knowing my luck, I put it in backwards lol